crates/common/src/result.rs

```diff
@@ -26,17 +26,19 @@ pub enum WasmError {
     Zome(String),
     /// somehow wasmer failed to compile machine code from wasm byte code
     Compile(String),
+
+    CallError(String),
 }
 
-impl From<std::num::TryFromIntError> for WasmError {
-    fn from(_: std::num::TryFromIntError) -> Self {
-        Self::PointerMap
+impl From<WasmError> for String {
+    fn from(e: WasmError) -> Self {
+        format!("{}", e)
```

Comment:
FYI if you use `#[error("some text: {0}")]` attrs over each variant, you get a Display impl for free.